### DISCUSSION TITLE
Target window not found Ahk v2

### DISCUSSION THREAD

**Discussion by Reaper-97** [Spawnova/ShinsOverlayClass#14] [Q&A]
Hi there, i want to use the OverlayClass for my Ahk v2 projects but even when i use the demo i get the error "Target window not found." when clicking out of the attached window and clicking back into it.

I tried to provide set alwaysontop to 0 also not active but then the overlay doesnt even show up anymore.
![AutoHotkey64_O3FC9B4pe8](https://github.com/user-attachments/assets/b85b40ba-6f2e-40b9-8461-819ddce58f6b)

**Comment by Spawnova** [comment]
Hi, can you try adding `DetectHiddenWindows True` to the top of your script and see if it still gives an error?

**Reply by Reaper-97** [reply]
```autohotkey
DetectHiddenWindows True
``` 
Worked instantly and solved my issue. Thanks a lot for the quick response and the awesome work you've done with the ahk classes.

**Reply by Spawnova** [reply]
Awesome, thanks for letting me know about this issue, I will try to incorporate a fix into the class later =P

**Reply by Spawnova** [reply]
Okay, sorry for the really late response, but I believe I solved this specific issue, so DetectHiddenWindows will no longer be needed
